test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/PublishIISCommandFacts.cs

```diff
@@ -18,12 +18,11 @@ private class Folders
         [Fact]
         public void PublishIIS_uses_default_values_if_options_not_specified()
         {
-            var webRoot = "wwwroot";
-            var folders = CreateTestDir("{}", webRoot);
+            var folders = CreateTestDir("{}");
 
-            new PublishIISCommand(folders.PublishOutput, folders.ProjectPath, null).Run();
+            new PublishIISCommand(folders.PublishOutput, folders.ProjectPath).Run();
 
-            var processPath = (string)GetPublishedWebConfig(folders.PublishOutput, webRoot)
+            var processPath = (string)GetPublishedWebConfig(folders.PublishOutput)
                 .Descendants("aspNetCore").Attributes("processPath").Single();
 
             Assert.Equal($@"..\projectDir.exe", processPath);
```

Comment:
Now that the web.config file would be a sibling with the executable...this should not have `..`?